### PR TITLE
[Improvement][Registry][Jdbc] Add jdbc registry config in helm charts

### PR DESCRIPTION
**Pull request by qingwli**

### CODE REVIEW

**Comment by rickchengx** [comment]
@qingwli please Run 'mvn spotless:apply' to fix these violations.

**Comment by qingwli** [comment]
@rickchengx  Done

**Comment by qingwli** [comment]
PTAL @kezhenxu94

File: deploy/kubernetes/dolphinscheduler/templates/_helpers.tpl [hunk 1]
```diff
@@ -244,28 +244,54 @@ Create a registry environment variables.
...
+- name: REGISTRY_HIKARI_CONFIG_JDBC_URL
+  value: {{ .Values.registryJdbc.hikariConfig.jdbcurl }}
+- name: REGISTRY_HIKARI_CONFIG_USERNAME
+  value: {{ .Values.registryJdbc.hikariConfig.username }}
+- name: REGISTRY_HIKARI_CONFIG_PASSWORD
+  valueFrom:
+    secretKeyRef:
+      name: {{ include "dolphinscheduler.fullname" . }}-externaldb
+      key: registry-password
+{{- end }}
```
Comment (kezhenxu94) [inline deploy/kubernetes/dolphinscheduler/templates/_helpers.tpl]: This doesn't make sense to me, you are using a single value `username` to determine whether all the other configurations should take effects or not, what if users want to change the `idelTimeout` but keep the default `username`?

Comment (qingwli) [reply deploy/kubernetes/dolphinscheduler/templates/_helpers.tpl]: Do you think we add a button like `useSameDatabase` to control this condition logic is better?

**Comment by qingwli** [comment]
PTAL @EricGao888 @rickchengx @Gallardot

**Review by Gallardot — APPROVED** [review]
LGTM

**Comment by qingwli** [comment]
@Gallardot  PLTA Again

**Comment by qingwli** [comment]
PTAL @Radeity @SbloodyS

**Review by Radeity — APPROVED** [review]
LGTM

**Pull request merged**